docs/product-user-guide/src/main/asciidoc/service-stop-remove-proc.adoc

```diff
@@ -0,0 +1,36 @@
+[id='service-stop-remove-proc_{context}']
+
+= Stopping and removing a service in {KIE_SERVER}
```

Comment:
For the scope of this adoc file, my jargon here was "Kie Container". Or could do also "decision service".